interfaces/acados_matlab_octave/ocp_compile_interface.m

```diff
@@ -112,82 +121,41 @@ function ocp_compile_interface(opts)
     LDFLAGS = 'LDFLAGS=$LDFLAGS -fopenmp';
 end
 
-% remove flag file, if present, before creating a new one
```

Comment:
I think these flag files were used to support the `auto` mode for the option `compile_interface`. Are you sure you want all of this?

Reply:
Yes, now the `auto` mode is done via the `link_libs.json`, thus I removed the flag file stuff here.

Reply:
I mean something else. The code used to put a flag file in de build folder, to remember for what qp_solver the ocp mex files were built. In `auto` mode, it read the name of that flag file, to see if it has to rebuild to ocp. At first sight, this flag file is not placed in the build folder anymore. Or can you just change the selected solver for the compiled ocp, without rebuilding the mex files?

Reply:
exaclty, now the Mex interface is built with all the external QP solvers specified in the `link_libs.json` file. So there is no need to recompile the interface when switching between different external QP solvers.

Reply:
I see. Perfect!